docs/zkapps/tutorials/09-recursion.mdx

```diff
@@ -56,15 +56,90 @@ You build recursive zkApps with [ZkProgram](/zkapps/o1js-reference/modules/Exper
 
 Proofs generated using a ZkProgram can be passed into zkApp smart contracts for them to verify recursively. They can even be passed recursively into their own functions for off-chain recursive composition.
 
-The following ZkProgram example code is provided in the [main.ts](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/09-recursion/src/main.ts) example file.
+The full source code for the following ZkProgram tutorial is provided in the [main.ts](https://github.com/o1-labs/docs2/blob/main/examples/zkapps/09-recursion/src/main.ts) directory on GitHub. While you're there, give the `/docs2` repository a star so that other zk developers can learn to build a zkApp!
 
-To create a ZkProgram, start with the `init()` method. 
 
-- For each method, declare the inputs it will receive. 
+## Prerequisites
+
+Ensure your environment meets the [Prerequisites](/zkapps/tutorials#prerequisites) for zkApp Developer Tutorials.
+
+In particular, make sure you have the zkApp CLI installed:
+
+```sh
+$ npm install -g zkapp-cli
+```
+
+## Create a new project
+
+Now that you have the tooling installed, you can start building your application.
+
+1. Create or change to a directory where you have write privileges.
+1. Now, create a project using the `zk project` command:
+
+```sh
+$ zk project 09-recursion
+```
+
+As you learned in earlier tutorials, the `zk project` command creates the `09-recursion` directory that contains the scaffolding for your project.
```

Comment:
```suggestion

    ```sh
    $ zk project 09-recursion
    ```
    
    As you learned in earlier tutorials, the `zk project` command creates the `09-recursion` directory that contains the scaffolding for your project.
```

Reply:
@LuffySama-Dev to keep enumeration of the Markdown numbered list, we indent the content for each step, see https://github.com/o1-labs/docs2/wiki/Markdown-Syntax#numbered-steps